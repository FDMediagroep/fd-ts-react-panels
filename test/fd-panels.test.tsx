import TestRenderer from 'react-test-renderer';
import React from 'react';
import Panel from '../src/fd-panels';

describe('FD Panel', () => {

    it('should render correctly', () => {
        let panel = TestRenderer.create(<Panel/>);
        expect(panel.toJSON()).toMatchSnapshot();
        panel = TestRenderer.create(<Panel boxShadow={true}/>);
        expect(panel.toJSON()).toMatchSnapshot();
    });
});
