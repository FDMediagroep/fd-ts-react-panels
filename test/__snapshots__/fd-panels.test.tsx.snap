// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FD Panel should render correctly 1`] = `
<div
  className="sc-bdVaJa bZwGnE"
/>
`;

exports[`FD Panel should render correctly 2`] = `
<div
  className="sc-bdVaJa kLEhFx"
/>
`;
